pages/game.js

```diff
@@ -20,7 +25,26 @@ export default class Index extends Component {
         isVictory: false,
     };
 
-    handleClick = () => {};
+    handleClick = tile => {
```

Comment:
s/tile/(tile) because it's not an implicit return here (with "()").

Reply:
They call me julienslint :trollface:

Reply:
If you want to trigger sedy, you have to add a final / to your sed command

Reply:
Unfortunately this is not part of my eslint rules :(

Reply:
No that's how prettier format it and it's fine